Shamaz/ViewController.swift

```diff
@@ -10,24 +10,44 @@ import UIKit
 
 class ViewController: UIViewController {
 
-    let test1 = "hello"
+    let futureStoryStartPhrases = ["What would you like to be doing", "What would you like to have for dinner", "Where would you like to be", "Who would you like to meet", "Who would you like to be with"]
     
-    let test2 = "world"
+    let pastStoryStartPhrases = ["What did you do", "Where were you", "What did you have for breakfast", "What did you have for dinner", "What did you wear", "Who did you talk to", "Who did you meet"]
     
+    let timeIndicators = ["1 day", "2 days", "3 days", "4 days", "5 days", "1 week", "3 weeks", "1 month", "3 months", "6 months", "1 year", "5 years", "10 years"]
```

Comment:
"a day", "a week", "a month" could be an improvement.
From the project requirement:

> The app will then generate a story prompt for the player to use. Some example story prompts could be: "Describe your day 2 days ago" Or "What would you like to be doing a month from now?". Phrases will either use the past or the future and include a time indicator (like "2 weeks ago" or "next year").